packages/vitest/src/runtime/inspector.ts

```diff
@@ -20,8 +20,14 @@ export function setupInspect(config: ResolvedConfig) {
     if (!isOpen) {
       inspector.open()
 
-      if (config.inspectBrk)
+      if (config.inspectBrk) {
+        const session = new inspector.Session()
+        session.connect()
+        session.post('Debugger.enable')
+
         inspector.waitForDebugger()
+        session.post('Debugger.pause')
```

Comment:
It would be great if we could actually call this on top of the first test file. I'll check later if that's doable easily.

Reply:
`Debugger.setBreakpointByUrl` should work. Below minimal repro:

```js
// entry.mjs
import inspector from "node:inspector";

const session = new inspector.Session();
session.connect();
session.post("Debugger.enable");

inspector.open();
inspector.waitForDebugger();
session.post("Debugger.setBreakpointByUrl", {
  lineNumber: 0,
  url: new URL("./target.mjs", import.meta.url).href,
});

console.log("Importing");
await import("./target.mjs");

console.log("Exit");
```

```js
// target.mjs
console.log("Line #1"); // Dev tools break here automatically
console.log("Line #2");
console.log("Line #3");
```